src/components/Picker.js

```diff
@@ -193,6 +193,21 @@ class Picker extends PureComponent {
                                 this.setState({isOpen: false});
                                 this.props.onBlur();
                             },
+
+                            // The following 2 handlers are specific to web (onChange overrides onValueChange on web)
+                            // They are used to make the picker indicator behave the same way as it does on native
+                            onMouseDown: () => {
+                                this.setState({isOpen: true});
+                            },
+                            onChange: (e) => {
+                                if (e.target.selectedIndex === undefined) {
+                                    return;
+                                }
+                                const index = e.target.selectedIndex;
+                                const value = e.target.options[index].value;
+                                this.onInputChange(value, index);
+                                this.setState({isOpen: false});
```

Comment:
It feels like we are sneaking in some platform dependent code here, but it's not clear which is the web or native implementation. Normally, [how we deal with that](https://github.com/expensify/app#platform-specific-file-extensions) is create a "base" component and then an `index.js` and `index.native.js` components. In a file like `Picker.js` we'd then just import the Picker but the platform dependent changes would be contained in the respective versions of the picker. Would this approach work here?